langserver/workspace_refs.go

```diff
@@ -217,7 +217,6 @@ func (h *LangHandler) externalRefsFromPkg(ctx context.Context, bctx *build.Conte
 				ContainerName: defContainerName,
 				PackageName:   defPkg.ImportPath,
 				Meta: map[string]interface{}{
-					"dir":    defPkg.Dir, // TODO: emit standard "file" instead?
```

Comment:
surprised you didn't need to update tests with this commit?

Edit: I see you updated in a later commit